libraries/AC_CustomControl/AC_CustomControl_LQR.cpp

```diff
@@ -0,0 +1,114 @@
+#include "AC_CustomControl_LQR.h"
+
+#if CUSTOMCONTROL_LQR_ENABLED
+
+#include <GCS_MAVLink/GCS.h>
+
+// table of user settable parameters
```

Comment:
where are these used?

Reply:
These parameters are not currently used. the lqr gains for each axis are hardcoded, and are currently set for a symmetrical 10 inch quadcopter. The details on how to calculate these params for different systems will be included in readme